ups/obs_base.table

```diff
@@ -1,14 +1,17 @@
-setupRequired(daf_base)
-setupRequired(log)
-setupRequired(daf_persistence)
-setupRequired(afw)
+# Python Middleware
 setupRequired(utils)
-setupRequired(astro_metadata_translator)
-setupRequired(geom)
 setupRequired(daf_butler)
 setupRequired(pipe_base)
-setupOptional(skymap)
 setupRequired(resources)
+setupRequired(astro_metadata_translator)
+setupRequired(pex_config)
+
+# LSST C++.
```

Comment:
Why the C++ annotation here?

Reply:
Because I'm trying to distinguish between packages that people can install from pypi vs packages that you can't.